app/src/main/java/com/mnbpdx/gardenbook/data/PlantRepository.kt

```diff
@@ -1,21 +1,30 @@
-import com.mnbpdx.gardenbook.data.PlantDao
+package com.mnbpdx.gardenbook.data
+
 import com.mnbpdx.gardenbook.model.Plant
 import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+// An interface that defines what the repository needs to do
+interface PlantRepository {
```

Comment:
I pulled out an `interface` for this repository so we could use it with Hilt, but idk it felt like good practice too